Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/setup.py

```diff
@@ -191,7 +191,7 @@ def git(cls, args, repository, additional_setup=None, hooks=None, **kwargs):
         # Pushing to http repositories is difficult, offer to change http checkouts to ssh
         http_remote = local.Git.HTTP_REMOTE.match(repository.url())
         if http_remote and not args.defaults and Terminal.choose(
-            "http based remotes will prompt for your password when pushing,\nwould you like to convert to a ssh remote?",
+            "http(s) based remotes will prompt for your password when pushing,\nwould you like to convert to a ssh remote?",
```

Comment:
Nit: can reword "for your password" = >"for your password every time"

It would be nice to clearly recommend ssh here as some (beginner) users might not know the difference or which is recommended approach.

Maybe:
convert to a ssh remote? => "convert to a ssh remote? (recommended)"

or maybe something like:
"http(s) based remotes will prompt for your password every time when pushing,\nIt is recommended to convert to a ssh remote, would you like to convert to a ssh remote?